### PR TITLE
divide and multiply

### PR DESCRIPTION
**Pull request by denispop9**
@evanhearne

### CODE REVIEW

**Review by evanhearne — APPROVED** [review]
Good boi.

**Comment by laurafitzgerald** [comment]
@denispop9 same here, I suggest trying to name the branch for what the change reflects. Remember that for the session on Monday 😉

File: calculator.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
 function addNumberToDisplay(number) {
-	var displayScreen = document.getElementById('displayScreen');
-	var displayScreenValue = displayScreen.value
-	displayScreen.value = displayScreenValue + number
+	const displayScreen = document.getElementById('displayScreen');
```
Comment (laurafitzgerald) [inline calculator.js:2]: @denispop9 what was the reason behind changing from var to const out of interest?

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <h1>Calculator Project</h1>
 <table>
 <tr>
-	<td><button>Del</button></td>
+	<td><button onclick="deleteFromDisplay('')">Del</button></td>
```
Comment (laurafitzgerald) [inline index.html:11]: @denispop9 I see that you are passing an empty value here, is it required?

**Pull request merged**